resources/views/pages/services-list.blade.php

```diff
@@ -4,20 +4,20 @@
 
 @section('content')
 
-    <div class="col-md-8 offset-md-2">
+    <div class="col-md-12 mt-4">
         <div class="row">
-            <div class="col-md-6 py-0">
-                <h3 class="mt-4">List of Services</h3>
+            <div class="col-md-8 py-0">
+                <h3 class="mt-4"><i data-feather="align-left"></i> List of Services</h3>
```

Comment:
Please retain these changes. If we are going to remove "List of Services" and change to "Lists" it will make the user see like they were hanging. Kuwang kung "Lists" ra ang ibutang.

Reply:
If I am the user it makes me feel like hanging. If we try to visualize the UI with the "List" in it instead of "List of Services" para nhu kuwang. mag.wonder ko dajun kng unsa ni nga list?